src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    if (robot[key] in invertedRobot) {
+      return null;
+    }
+
+    invertedRobot[robot[key]] = key;
```

Comment:
Consider using 'Object.hasOwnProperty()' to check if the property exists on 'invertedRobot'. Directly using 'in' might lead to unexpected results if the object has inherited properties.